operator/data/profiles/default.yaml

```diff
@@ -181,7 +181,32 @@ spec:
             maxUnavailable: "25%"
 
     egressGateways:
-
+    - name: istio-egressgateway
+      enabled: false
+      k8s:
```

Comment:
-1 to this. Why do we need to copy everything over? its already defined in the templates. Its so hard to figure out what I am going to get when I install with istioctl now because there are so many different configs

Reply:
I would update this part to keep only the settings we expose as params in the template

Reply:
actually only the scaleTargetRef is hardcoded in the template, however we can not drop this part from the profile because of existing issue: https://github.com/istio/istio/issues/20143